### PR TITLE
fix(lambda-tiler): send 408 timeout response rather than timing out.

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: packages/lambda-tiler/src/index.ts [hunk 1]
```diff
@@ -20,18 +20,17 @@ import { St } from './util/source.tracer.js';
...
+handler.router.timeoutEarlyMs = 1_000;
+
 handler.router.hook('request', (req) => {
   req.set('name', 'LambdaTiler');
 
   // Reset the request tracing before every request
   St.reset();
 });
 
-let totalRequests = 0;
```
Comment (blacha) [inline packages/lambda-tiler/src/index.ts:30]: linzjs/lambda now counts requests.

File: packages/lambda-tiler/src/index.ts [hunk 2]
```diff
@@ -20,18 +20,17 @@ import { St } from './util/source.tracer.js';
 
 export const handler = lf.http(LogConfig.get());
 
+/** If the request takes too long, respond with a 408 timeout when there is approx 1 second remaining */
+handler.router.timeoutEarlyMs = 1_000;
```
Comment (Wentao-Kuang) [inline packages/lambda-tiler/src/index.ts:24]: Is 1 second enough for attribution?

Comment (blacha) [reply packages/lambda-tiler/src/index.ts:24]: its `lambda.maxDuration - 1_000` so its 59,000ms since we have a default duration of 1 minute.

Comment (Wentao-Kuang) [reply packages/lambda-tiler/src/index.ts:24]: Haha, I see.

**Review by Wentao-Kuang — APPROVED** [review]

**Pull request merged**